packages/data-point/lib/entity-types/integrations.test.js

```diff
@@ -222,34 +223,39 @@ test('Model Entity Instance', () => {
 })
 
 describe('trace feature', () => {
-  test('trace via options parameter', () => {
-    const consoleTime = console.time
-    const consoleTimeEnd = console.timeEnd
+  let mockDateNow
+  let mockWriteFileP
+  afterEach(() => {
+    mockDateNow.mockRestore()
+    mockWriteFileP.mockRestore()
```

Comment:
these variables are only used in one test, so is it necessary to put them in `afterEach` / describe scope?

Reply:
yes, because if the test fail, the mock restoration will get executed regardless